config/config.go

```diff
@@ -53,14 +52,15 @@ func getOrCreatePath(loc string, perm os.FileMode, isDir bool) error {
 
 func GetDefaultConfigHome() string {
 	var configHome string
-	if runtime.GOOS == "darwin" {
-		configHome = os.Getenv("HOME")
-	} else if runtime.GOOS == "linux" {
-		configHome = os.Getenv("XDG_CONFIG_HOME")
-		if configHome == "" {
-			configHome = os.Getenv("HOME")
-		}
+	var isPresent bool
+
+	configHome, isPresent = os.LookupEnv("XDG_CONFIG_HOME")
+	if isPresent {
```

Comment:
I think this might affect our existing linux users if they already have everything stored in `$HOME/.corgi`, and that's why I added the runtime sys check

Reply:
I don't think its a problem here. If a macOS user have set the `XDG_CONFIG_HOME` variable he will expect it to work as in Linux. In fact, chances are that `$HOME/.corgi` will be a symlink to `$XDG_CONFIG_HOME/corgi`. So this PR wil not affect his workflow.

I think that for the users of macOS that have set `XDG_CONFIG_HOME` and have let *corgi* store its data in `$HOME/.corgi`, a message in the release changelog should suffice.
You're still not in v1 yet, so breaking changes should be expected.